src/main/java/pl/llp/aircasting/screens/sessions/shareSession/ShareSessionActivity.java

```diff
@@ -67,6 +51,7 @@
     private Session session;
     private ShareSessionViewMvcImpl mShareSessionView;
     private CharSequence mSelectedSensor;
+    private CharSequence emailAddres;
```

Comment:
Typo, should be `EmailAddress`. Also, you could follow the instance variable android convention here and call it `mEmailAddress`